core/templates/dev/head/pages/topic_editor/TopicEditorNavbarDirective.js

```diff
@@ -58,7 +58,11 @@ oppia.directive('topicEditorNavbar', [
               function() {
                 $scope.topicRights.markTopicAsPublished();
                 TopicEditorStateService.setTopicRights($scope.topicRights);
-              });
+              }).then(function(){
+              var successToast = ('Topic Published.');
```

Comment:
Btw, here and elsewhere, you don't need the parentheses surrounding the successs message. Those are only useful when the message spans multiple lines (like line 137 in SkillsListDirective.js).

Reply:
Then how do I know which files it should be removed from

Reply:
Done